api/Document.json

```diff
@@ -10150,7 +10150,7 @@
               }
             ],
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
Yeah, the true there wasn't plausible, given that Edge didn't have `document.timeline`.